apps/prairielearn/src/middlewares/authzCourseOrInstance.js

```diff
@@ -115,6 +115,8 @@ export async function authzCourseOrInstance(req, res) {
     res.locals,
   );
 
+  res.locals.bot_grading_enabled = await features.enabledFromLocals('bot-grading', res.locals);
+
```

Comment:
I don't think this needs to be in the middleware, it could go in whatever page needs the information.